tests/tests.py

```diff
@@ -11,7 +11,7 @@
 class DNSTest(unittest.TestCase):
 
     def setUp(self):
-        self.channel = pycares.Channel(timeout=1.0, tries=1)
+        self.channel = pycares.Channel(timeout=5.0, tries=1)
```

Comment:
I see. Any chance we can test against some highly available server like Google? I know they are not going away anytime soon ;-)